ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
If chunkSize is not incremented for that case, as currently written, there will be rows at the end of the table which are not processed.

Consider the case for nrows = 10,001 

currently then, chunkSize = 11 and nchunks = 910. In this case, the loop covers all rows.

Without the if block for this case, nchunks is 1000 and chunkSize is 10. In this case, the last row will not be processed.